configure.py

```diff
@@ -990,8 +990,6 @@ def main():
                 'with_gcp_support', False, 'gcp')
   set_build_var(environ_cp, 'TF_NEED_HDFS', 'Hadoop File System',
                 'with_hdfs_support', False, 'hdfs')
-  set_build_var(environ_cp, 'TF_NEED_S3', 'Amazon S3 File System',
-                'with_s3_support', False, 's3')
```

Comment:
What happened to this commit? The commit message is totally irrelevant.

Reply:
Also is this S3 build var removal on purpose?

Reply:
@xiejw might know more. But I think it was re-added in a followup change. If you look at all complete changes it is still there.

Reply:
Got it. Thanks!